packages/core/src/components/CopyToClipboard/CopyToClipboard.tsx

```diff
@@ -9,7 +9,7 @@ import { useTimeout } from 'react-use-timeout';
 
 const StyledAssignmentIcon = styled(({ invertColor, ...rest }) => <AssignmentIcon {...rest} />)(
   ({ theme, invertColor }) => `
-  color: ${invertColor ? theme.palette.common.white : theme.palette.text.secondary};
+  color: ${invertColor ? theme.palette.common.white : '#757575'};
```

Comment:
The #757575 color used for non-inverted mode doesn't look good in dark mode. Settings --> Profiles is an example where the non-inverted color is used:

![image](https://user-images.githubusercontent.com/339312/207395099-fd67df74-e444-42db-95f6-94631424dae2.png)